.github/workflows/open_caller_prs.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+"""
+Whenever a new reusable workflow is created or updated, we want to open a PR in every repository from which we want to call it.
```

Comment:
I don't think it will always be the case that we want to strictly describe the workflows (using caller-templates) in which a reusable workflow is used. As the git ref of a reusable workflow can be specified e.g. as `main`, consuming workflows will get updates.

This change seems to be about updating the _context_ in which shared workflows are used which might vary depending on the shared workflow, though in the case of the stale PR workflow this probably does make sense.